recipe/meta.yaml

```diff
@@ -58,23 +58,27 @@ outputs:
       - lib/pkgconfig/ode.pc                                         # [unix]
       - lib/cmake/ode-{{ soversion }}                                # [unix]
       - bin/ode-config                                               # [unix]
+      - include/ode                                                  # [unix]
       - Library/lib/ode_double.lib                                   # [win]
       - Library/bin/ode_double.dll                                   # [win]
       - Library/lib/pkgconfig/ode.pc                                 # [win]
       - Library/lib/cmake/ode-{{ soversion }}                        # [win]
-      - include/ode
+      - Library/include/ode                                          # [win]
+ 
+     
     test:
       commands:
         - test -f $PREFIX/lib/libode${SHLIB_EXT}                     # [unix]
         - test -f $PREFIX/lib/libode${SHLIB_EXT}.{{ soversion }}     # [linux]
         - test -f $PREFIX/lib/libode.{{ soversion }}${SHLIB_EXT}     # [osx]
-        - test -d $PREFIX/include/ode                                # [unix]
+        - test -d $PREFIX/include/ode/ode.h                          # [unix]
```

Comment:
that should fail, shouldn't it be a test -f?

Reply:
Thanks, fixed in https://github.com/conda-forge/libode-feedstock/pull/7/commits/be22e734d122829aec6ea8b2bd64b9084480a0f8 .